README.adoc

```diff
@@ -236,27 +236,27 @@ target URL as an argument and returns a JSON object that contains HATEOAS links.
 ----
 
 InventoryUtil.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=comment']
 ----
-include::finish/src/main/java/io/openliberty/guides/microprofile/util/InventoryUtil.java[tags=!comment]
+include::finish/src/main/java/io/openliberty/guides/microprofile/util/InventoryUtil.java[]
 ----
 
-The [hotspot=32-37 file=1]`buildHostJson` helper method builds in the `InventoryUtil` class. It
-creates a JSON object that contains the [hotspot=34 file=1]`hostname` of your system and the [hotspot=35 file=1]`_links` array, 
-which is generated separately in the [hotspot=39-52 file=1]`buildLinksForHost` helper method.
+The [hotspot=buildHostJson file=1]`buildHostJson` helper method builds in the `InventoryUtil` class. It
+creates a JSON object that contains the [hotspot=hostname file=1]`hostname` of your system and the [hotspot=links file=1]`_links` array, 
+which is generated separately in the [hotspot=buildLinksForHost file=1]`buildLinksForHost` helper method.
 
 This helper accepts a hostname and a target URL as arguments. The helper builds a link that points to the
-inventory entry with a [hotspot=45 file=1]`self` relationship and also builds a link that points to the `system` service with
-a [hotspot=49 file=1]`properties` relationship:
+inventory entry with a [hotspot=self file=1]`self` relationship and also builds a link that points to the `system` service with
+a [hotspot=properties file=1]`properties` relationship:
 
 * \http://localhost:9080/inventory/hosts/<hostname>
 * \http://<hostname>:9080/system/properties
```

Comment:
These should be hyperlinked. Otherwise if it's not an accessible URL, enclose them in `backticks` instead of a hyperlink

Reply:
Not intended to be links. Notice `<hostname>` for user input.